helm-charts/charts/kube-starrocks/templates/starrocks-operator/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
I am thinking that the namespace should be respect of `helm -n` option, if we specify namespace in template, will it still respect the `-n <namespace>` issued by `helm`?

Reply:
There are a few things that need to be clarified:

1. By default, the Operator is deployed to `~/.kube/config` in the Namespace specified by `context.namespace`.
2. If you execute `helm install -n <namespace>`, the Operator will be installed in that Namespace.
3. If `.Values.starrocksOperator.namespaceOverride` is specified, it takes the highest priority.

Creating helm-charts/charts/kube-starrocks/templates/starrocks-operator/namespace.yaml ensures the existence of the namespace.

Reply:
ok. remember to add cases to cover this scenario, ensure when user specify `-n <namespace>`. it will take effect.

if `.Values.starrocksOperator.namespaceOverride`, our operator will be deployed to `default` namespace?